test/print.jl

```diff
@@ -197,18 +197,39 @@ end
         constr_ge = @NLconstraint(model, sin(x) >= 1)
         constr_eq = @NLconstraint(model, sin(x) == 1)
         constr_range = @NLconstraint(model, 0 <= sin(x) <= 1)
+        constr_exponent_1 = @NLconstraint(model, x^4 <= 1)
+        constr_exponent_2 = @NLconstraint(model, x^40.23 <= 1)
+        constr_exponent_3 = @NLconstraint(model, x^4 + x^3 + x^2 <= 1)
+        constr_exponent_4 = @NLconstraint(model, x^-4 + x^-3 + x^-2 <= 1)
+        constr_exponent_5 = @NLconstraint(model, x^(x * x) <= 1)
+        constr_exponent_6 = @NLconstraint(model, x^(2x) <= 1)
+        constr_exponent_7 = @NLconstraint(model, x^(2 * 5) <= 1)
 
         io_test(REPLMode, constr_le, "sin(x) - 1.0 $le 0")
         io_test(REPLMode, constr_ge, "sin(x) - 1.0 $ge 0")
         io_test(REPLMode, constr_eq, "sin(x) - 1.0 $eq 0")
         # Note: This is inconsistent with the "x in [-1, 1]" printing for
         # regular constraints.
         io_test(REPLMode, constr_range, "0 $le sin(x) $le 1")
+        io_test(REPLMode, constr_exponent_1, "x ^ 4.0 - 1.0 $le 0")
+        io_test(REPLMode, constr_exponent_2, "x ^ 40.23 - 1.0 $le 0")
+        io_test(REPLMode, constr_exponent_3, "(x ^ 4.0 + x ^ 3.0 + x ^ 2.0) - 1.0 $le 0")
+        io_test(REPLMode, constr_exponent_4, "(x ^ -4.0 + x ^ -3.0 + x ^ -2.0) - 1.0 $le 0")
+        io_test(REPLMode, constr_exponent_5, "x ^ (x * x) - 1.0 ≤ 0")
```

Comment:
Hard-coding the inequality symbol causes CI failures: https://github.com/jump-dev/JuMP.jl/pull/2420/checks?check_run_id=1641836693#step:6:199.

Interpolate the symbol instead as above.